mllib/src/test/scala/org/apache/spark/mllib/recommendation/MatrixFactorizationModelSuite.scala

```diff
@@ -72,6 +72,22 @@ class MatrixFactorizationModelSuite extends SparkFunSuite with MLlibTestSparkCon
     }
   }
 
+  test("invalid user and product") {
+    val model = new MatrixFactorizationModel(rank, userFeatures, prodFeatures)
+    assert(intercept[IllegalArgumentException]  {
```

Comment:
I don't think it's necessary to assert about the exact message string. Maybe assert it contains the user ID. But just checking for the exception also seems close enough.

Reply:
Thanks for the review. Done.